Sources/NIO/Bootstrap.swift

```diff
@@ -967,3 +999,146 @@ public final class NIOPipeBootstrap {
         }
     }
 }
+
+/// An channel option which can be applied to bootstrap using shorthand notation.
+/// - See: ServerBootstrap.serverOptions(_ options: [ShorthandBootstrapOption])
+public struct ShorthandServerBootstrapOption {
+    private let data: ShorthandServerOption
+    
+    private init(_ data: ShorthandServerOption) {
+        self.data = data
+    }
+    
+    /// Apply the contained option to the supplied ServerBootstrap
+    /// - Parameter serverBootstrap: bootstrap to apply this option to.
+    /// - Returns: the modified bootstrap (currently the same one mutated)
+    func applyOption(to serverBootstrap : ServerBootstrap) {
+        data.applyOption(to: serverBootstrap)
+    }
+    
+    private enum ShorthandServerOption {
+        case ReuseAddr(Bool)
+        case AutoRead(Bool)
+        case Backlog(Int32)
+        
+        func applyOption(to serverBootstrap: ServerBootstrap) {
+            switch self {
+                case .AutoRead(let value):
+                    let _ = serverBootstrap.serverChannelOption(ChannelOptions.autoRead, value: value)
```

Comment:
The standard indentation pattern for switch statements in Swift is:

```swift
switch x {
case .y:
    x.whatever()
case .z:
    x.whateverElse()
}
```

Reply:
Corrected to standard indentation.